operator/cmd/mesh/install.go

```diff
@@ -297,6 +297,7 @@ func savedIOPName(iop *v1alpha12.IstioOperator) string {
 // nolint: interfacer
 func detectIstioVersionDiff(p Printer, tag string, ns string, kubeClient kube.CLIClient, setFlags []string) error {
 	warnMarker := color.New(color.FgYellow).Add(color.Italic).Sprint("WARNING:")
+	revision := manifest.GetValueForSetFlag(setFlags, "revision")
```

Comment:
if they don't set this, will it be comparing `default != ""`?

Reply:
Fixed, thanks.

Reply:
Also I noticed that using the set flag was not quite accurate, since the revision may have been set in the overlaid file instead of the set flag.